src/Gelf/Logger.php

```diff
@@ -140,6 +141,33 @@ protected function initMessage($level, $message, $context)
     }
 
     /**
+     * Initializes context array, ensuring all values are string-safe
+     *
+     * @param array $context
+     * @return array
+     */
+    protected function initContext($context)
+    {
+        foreach ($context as $key => &$value) {
+            switch (gettype($value)) {
+                case 'array':
+                case 'boolean':
+                    $value = json_encode($value);
+                    break;
+                case 'object':
+                    if (method_exists($value, '__toString')) {
+                        $value = (string)$value;
+                    } else {
+                        $value = '[object]';
```

Comment:
Hm. One more idea: `$value = "[object (" . get_class($value) . ")]";` -- What do you think?